api/src/http.rs

```diff
@@ -420,9 +420,9 @@ impl Default for ProxyConfig {
 /// Configuration for mirror.
 #[derive(Clone, Deserialize, Serialize, Debug)]
 pub struct MirrorConfig {
-    /// Mirror url host.
+    /// Mirror server URL, for example http://127.0.0.1:65001
     pub host: String,
-    /// Request headers for mirror server.
+    /// HTTP request headers to be passed to mirror server
```

Comment:
Is it valid to pass in an empty headers hashmap?

Reply:
It will return error. 
Fixed now.